overlay.mk

```diff
@@ -110,6 +109,8 @@ PRODUCT_PACKAGES += \
 	treble-overlay-xiaomi-redminote7 \
 	treble-overlay-xiaomi-redminote8pro \
 	treble-overlay-xiaomi-redmis2 \
+ 	treble-overlay-htc-exodus1 \
```

Comment:
uh what

Reply:
No idea about the htc-exodus itself, I just adapted the overlay.mk such that I survived your test script... Sorry in case I introduced some unintended glitch!